scripts/sell_unused_slots.py

```diff
@@ -80,6 +80,10 @@ def from_response(cls, response: JSON) -> 'Lambda':
         return cls(name=response['FunctionName'],
                    slot_location=env['AZUL_TDR_SOURCE_LOCATION'])
 
+    def __attrs_post_init__(self):
+        allowed_locations = config.tdr_allowed_source_locations
+        assert self.slot_location in allowed_locations, self.slot_location
+
 
 class ReindexDetector:
```

Comment:
Consider making the location an instance attribute of this class and looping over the locations on the outside.